crates/bevy_ecs/src/schedule/schedule.rs

```diff
@@ -1322,7 +1322,21 @@ impl ScheduleGraph {
 impl ScheduleGraph {
     fn get_node_name(&self, id: &NodeId) -> String {
         let mut name = match id {
-            NodeId::System(_) => self.systems[id.index()].get().unwrap().name().to_string(),
+            NodeId::System(_) => {
+                let name = self.systems[id.index()].get().unwrap().name().to_string();
+                if self.settings.report_sets {
+                    let sets = self.names_of_sets_containing_node(id);
+                    if sets.is_empty() {
+                        format!("{name} (no sets)")
```

Comment:
```suggestion
                        name
```

I don't think there's much value in explicitly stating that the system doesn't belong to any sets -- feel free to ignore if you disagree though.